bsnes/sfc/cpu/serialization.cpp

```diff
@@ -45,6 +45,12 @@ auto CPU::serialize(serializer& s) -> void {
 
   s.integer(status.autoJoypadCounter);
 
+  s.integer(status.autoJoypadPort0);
+  s.integer(status.autoJoypadPort1);
+
+  s.boolean(status.cpuLatch);
+  s.boolean(status.autoJoypadLatch);
+
```

Comment:
If we change what's being serialised, we should also bump the serialisation version.

Reply:
You're right, and that's one thing I was contemplating while making the serialization and struct changes. But I looked at the last time serialization was changed (34af95da9ea8ec2b5957f62ff3265c09628192fc) and the version wasn't bumped after that either. Should I just bump it to something like `115.1`?